display_list/skia/dl_sk_dispatcher.cc

```diff
@@ -254,12 +254,15 @@ void DlSkCanvasDispatcher::drawAtlas(const sk_sp<DlImage> atlas,
     return;
   }
   std::vector<SkColor> sk_colors;
-  sk_colors.reserve(count);
-  for (int i = 0; i < count; ++i) {
-    sk_colors.push_back(colors[i].argb());
+  if (colors != nullptr) {
+    sk_colors.reserve(count);
+    for (int i = 0; i < count; ++i) {
+      sk_colors.push_back(colors[i].argb());
+    }
   }
-  canvas_->drawAtlas(skia_atlas.get(), xform, ToSkRects(tex), sk_colors.data(),
-                     count, ToSk(mode), ToSk(sampling), ToSkRect(cullRect),
+  canvas_->drawAtlas(skia_atlas.get(), xform, ToSkRects(tex),
+                     colors == nullptr ? nullptr : sk_colors.data(), count,
```

Comment:
```suggestion
                     sk_colors.empty() ? nullptr : sk_colors.data(), count,
```

This is a more direct condition =)

Reply:
Done